package.json

```diff
@@ -1,91 +1,42 @@
 {
-  "name": "graphql-hooks",
-  "version": "3.2.2",
-  "description": "Graphql Hooks",
-  "main": "lib/graphql-hooks.js",
-  "module": "es/graphql-hooks.js",
-  "unpkg": "dist/graphql-hooks.min.js",
+  "name": "root",
+  "private": true,
   "scripts": {
     "test": "jest",
-    "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "eslint": "eslint src test",
+    "postinstall": "lerna bootstrap --no-ci",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "eslint": "eslint packages/*/src packages/*/test",
     "prettier": "prettier src/**/*.js test/**/*.js --write",
-    "build": "rollup -c",
-    "prepublishOnly": "npm run build"
-  },
-  "files": [
-    "dist",
-    "es",
-    "lib"
-  ],
-  "keywords": [
-    "graphql",
-    "hooks",
-    "graphql-hooks",
-    "gql",
-    "gql-hooks",
-    "react-graphql-hooks",
-    "react",
-    "apollo"
-  ],
-  "author": "Brian Mullan <bmullan91@gmail.com>",
-  "license": "Apache-2.0",
-  "peerDependencies": {
-    "react": "^16.8.3"
+    "release": "lerna publish"
   },
   "devDependencies": {
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
     "babel-eslint": "10.0.1",
     "babel-jest": "24.1.0",
-    "eslint": "5.15.0",
+    "coveralls": "3.0.3",
     "eslint-config-prettier": "4.1.0",
     "eslint-plugin-prettier": "3.0.1",
-    "eslint-plugin-react": "7.12.4",
     "eslint-plugin-react-hooks": "1.4.0",
+    "eslint-plugin-react": "7.12.4",
+    "eslint": "5.15.0",
     "husky": "1.3.1",
     "jest": "24.1.0",
-    "jest-fetch-mock": "2.1.1",
+    "lerna": "3.13.1",
     "lint-staged": "8.1.5",
-    "prettier": "1.16.4",
-    "react": "16.8.3",
-    "react-dom": "16.8.3",
-    "react-hooks-testing-library": "0.3.6",
-    "react-testing-library": "6.0.0",
-    "rollup": "1.4.1",
-    "rollup-plugin-babel": "4.3.2",
-    "rollup-plugin-node-resolve": "4.0.1",
-    "rollup-plugin-size-snapshot": "0.8.0",
-    "rollup-plugin-terser": "4.0.4"
+    "prettier": "1.16.4"
   },
   "lint-staged": {
-    "./src/**/*.js": [
-      "eslint src test",
-      "prettier src/**/*.js test/**/*.js --write",
+    "./packages/*/src/**/*.js": [
```

Comment:
On seeing this again, think this needs to be updated to `./packages/*/{src,test}/**/*.{js,md}` or the likes.

Reply:
Will `eslint` panic if we pass it a `.md` file?

side note: I had a question on the lint-staged setup that you might able to answer @jh3y. Why do we pass files to `prettier` and `eslint` in the commands? I think the idea is that it only runs the commands against the staged files, which should be passed to it by `lint-staged`.

Reply:
Tbh, those globs would need to be played with but `eslint` won't "panic". It will likely throw a parsing error without any extra configuration. It won't crash `eslint` though.

Good point. Could be reduced to `eslint` followed by `prettier --write` I imagine.

Reply:
I've updated the lint-staged config now, i didn't set it up to run against md files though as eslint couldn't deal with them.